config/locales/en/transition_landing_page.yml

```
@@ -2,10 +2,12 @@ en:
   transition_landing_page:
     meta_title: The UK transition
     meta_description: The UK's transition period after Brexit comes to an end this year. Check the new rules from January 2021 and take action now.
-    page_header: The transition period ends in December
+    page_header: "The UK transition:<br/>time is running out"
     page_header_explainer: |
-      The UK has left the EU, and the transition period after Brexit comes to an end this year. Check the new rules from January 2021 and take action now.
-    take_action_title: Make sure you're ready
+      <span class="landing-page__intro-highlight">There are new rules</span> for businesses and citizens from 1 January 2021.
```

This should be a `strong` tag, since it's being highlighted.
```suggestion
      <strong class="landing-page__intro-highlight">There are new rules</strong> for businesses and citizens from 1 January 2021.
```

---

Thanks